cl/scrapers/tasks.py

```diff
@@ -296,7 +300,7 @@ def extract_recap_pdf_base(
 
         rd.plain_text, _ = anonymize(content)
         # Do not do indexing here. Creates race condition in celery.
-        rd.save(index=False, do_extraction=False)
+        await sync_to_async(rd.save)(index=False, do_extraction=False)
```

Comment:
Why not use `.asave()` here?

Reply:
Doesn't seem to work:
```python
  File "/opt/courtlistener/cl/scrapers/tasks.py", line 237, in extract_recap_pdf
    return async_to_sync(extract_recap_pdf_base)(
      ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 240, in __call__
    return call_result.result()
    ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
    ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
    ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 306, in main_wrap
    result = await self.awaitable(*args, **kwargs)
    ^^^^^^^^^^^^^^^^^
  File "/opt/courtlistener/cl/scrapers/tasks.py", line 303, in extract_recap_pdf_base
    await rd.asave(index=False, do_extraction=False)
    ^^^^^^^^^^^^^^^^^
TypeError: Model.asave() got an unexpected keyword argument 'index'
```

Reply:
Interesting. Must be because we override the save() function, here: https://github.com/freelawproject/courtlistener/blob/b1fa4b21117e963f34349908b342dd4a10072ab0/cl/search/models.py#L1378-L1385 ? 

Looks like this might be a way forward: https://github.com/jjorissen52/epic-rpg-discord-helper/blob/9e1108b1e2cad81a00db4e28fc63f7f26ae12942/epic/models.py#L458

Reply:
Sure, we can do something like that instead, I've gone ahead and used [the upstream django style for this](https://github.com/django/django/blob/4.2.2/django/db/models/base.py#L823-L831).